### PR TITLE
Ignore plans folder and any subfolder

### PR DESCRIPTION
**Pull request by cyberious**

### CODE REVIEW

File: lib/puppetlabs_spec_helper/rake_tasks.rb [hunk 1]
```diff
@@ -220,7 +220,7 @@
 PuppetSyntax.exclude_paths << 'spec/fixtures/**/*'
 PuppetSyntax.exclude_paths << 'pkg/**/*'
 PuppetSyntax.exclude_paths << 'vendor/**/*'
```
Comment (alexjfisher) [inline lib/puppetlabs_spec_helper/rake_tasks.rb]: Whilst you're at it, could you also add `.vendor/**/*`?  This is already in the puppet-lint ignore_paths, but missing here.

**Comment by lionce** [comment]
Hello @cyberious ,

Would you please check the comments? We appreciate your help and support! Thanks!

Cheers!

File: lib/puppetlabs_spec_helper/rake_tasks.rb (hunk 1 above)
Comment (cyberious) [reply lib/puppetlabs_spec_helper/rake_tasks.rb]: Hi @alexjfisher, I see your point, why would we have Puppet Code in a `.vendor` repo, every time I have used a `.vendor` it has been for Ruby Gems. I am fine making this change but for Puppet Syntax I am scratching my head as to why we would hide modules in `.vendor`

**Comment by cyberious** [comment]
@lionce commented above and added change

**Comment by lionce** [comment]
Hello @cyberious ,

Thank you! Appreciate your help and patience.

Cheers!

**Pull request merged**